nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -1003,8 +1026,14 @@ in
         source = ''"''${SHARED_DIR:-$TMPDIR/xchg}"'';
         target = "/tmp/shared";
       };
+      certs = mkIf cfg.useHostCerts {
+        source = ''"$TMPDIR"/certs'';
+        target = "/etc/ssl/certs";
+      };
     };
 
+    security.pki.enable = mkIf cfg.useHostCerts false;
```

Comment:
I'm a dummy when it comes to security, so this is a basic question: what's the implication on the guest VM if this is false? Will there be no certificates at all, and does that significantly limit the usability?

Reply:
Instead of automatically disabling `security.pki` I think it should be treated as an error if the two options conflict.  In other words, something like:

```nix
{ assertions = [
    { assertion = !(config.services.pki.enable && config.virtualisation.useHostCerts);
      message = "…";
    }
  ];
}
```

Reply:
I don't think PKI is something you can disable. The packages you install will rely on it anyway. They generally can't even be configured to work without it, so the idea that `security.pki.enable = false;` would disable this crucial security feature is not realistic, and not something anyone would want in real life systems. Probably not even in tests.

How about `security.pki.etc.enableCACert` or something like that?

I think it would also make more sense for the `ca.nix` module to be responsible for handling this use case. That way it can't be accidentally forgotten about when `ca.nix` is inevitably changed in a way that's not compatible with mounted files.

Reply:
Besides mounted files, this module could support mutable ca files, which is similar but does not require virtualization or any kind of mount.
In both cases, the added value of this module is the knowledge of which files should be symlinked to the certificate file; whether that's a file that's composed from n immutable sources, or a single mutable or mounted file.

Reply:
I've updated the name of the option to `security.pki.installCACerts`. I like the idea of supporting mutable CA files, however I would prefer not to do it in this PR as I think it would be out of the scope of this PR and potentially quite complex.

Reply:
Yeah that's fine. I mentioned it as an example use case that might come later.